<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tunnel</class>
 <widget class="QDialog" name="tunnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tunnel Configuration</string>
  </property>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="key_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="host_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="host_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Host/IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tunnel_entries">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>721</width>
     <height>281</height>
    </rect>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Host</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Port</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Key</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32765</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name_line</tabstop>
  <tabstop>port_line</tabstop>
  <tabstop>host_line</tabstop>
  <tabstop>user_line</tabstop>
  <tabstop>password_line</tabstop>
  <tabstop>key_line</tabstop>
  <tabstop>browse_button</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>tunnel_entries</tabstop>
  <tabstop>confirm_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
